src/pages/resultsView/ResultsViewPageStore.ts

```diff
@@ -1307,9 +1307,10 @@ export class ResultsViewPageStore {
     readonly molecularData = remoteData<NumericGeneMolecularData[]>({
         await: () => [this.sampleKeyToSample, this.molecularData_preload],
         invoke: () => {
+            const sampleKeys = this.sampleKeyToSample.result!;
```

Comment:
I realized that since this is potentially called many times it would be better not to repeatedly access it

Reply:
good catch